server/Arcadia.Assistant.Web/Download/AppCenterBuildDownloadModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Arcadia.Assistant.Web.Download
+{
+    using System;
+
+    public class AppCenterBuildDownloadModel
+    {
+        public string Uri { get; set; }
+
+        public DateTime BuildDate { get; set; }
```

Comment:
DateTime or DateTimeOffset?

Reply:
"finishTime": "2018-12-18T12:48:15.7096069Z"

Reply:
it has Z -> DateTimeOffset

Reply:
Changed.